src/Bridges/Constraint/quad_to_soc.jl

```diff
@@ -136,23 +156,7 @@ function matrix_from_quadratic_terms(
         end
     end
     # Duplicate terms are summed together in `sparse`
-    Q = sparse(I, J, V, n, n)
-    return Q, index_to_variable_map
-end
-
-function matrix_to_vector_affine_terms(
-    L::SparseMatrixCSC{T},
-    p::Vector,
-    index_to_variable_map::Dict{Int,MOI.VariableIndex},
-) where {T}
-    # We know that L × L' ≈ Q[p, p] hence (L × L')[i, :] ≈ Q[p[i], p]
-    # We precompute the map to avoid having to do a dictionary lookup for every
-    # term
-    variable = map(i -> index_to_variable_map[p[i]], 1:size(L, 1))
-    function term(i::Integer, j::Integer, v::T)
-        return MOI.VectorAffineTerm(j, MOI.ScalarAffineTerm(v, variable[i]))
-    end
-    return map(ijv -> term(ijv...), zip(findnz(L)...))
```

Comment:
Is this what Julia couldn't infer ? I wouldn't have guessed. It can infer the result of `findnz(L)` so the type of the elements of the `zip`, etc...

Reply:
Hmm. I just tried in a new REPL and it seemed to work. So maybe I had something weird with Revise. The new way is quicker anyway because we avoid the multiple copies of the list of terms.